CONTRIBUTING.md

```diff
@@ -41,7 +41,20 @@ This is a rough outline of how to prepare a contribution:
 - Make sure any new code files have a license header.
 - Submit a pull request to the original repository.
 
-Thanks for your contributions!
+# Acceptance policy
+
+These things will make a PR more likely to be accepted:
+
+ * a well-described requirement
+ * tests for new code
+ * tests for old code!
+ * new code follows the conventions in old code
+
```

Comment:
Even though it's got its own section below, maybe mention a well-formatted commit message per the Commit Message section, just to re-inforce it.  Too many people using GitHub write awful commit messages.